client-shell/README.md

```diff
@@ -0,0 +1,108 @@
+<!---
+Copyright 2018-2021 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--->
+
+# Palisade Shell
+
+The Palisade Shell provides a CLI REPL for access to a cluster and its resources.
+
+> :information_source:
+In the case of the [Palisade examples](https://github.com/gchq/Palisade-examples/), the `context` is limited to just a `purpose` - one of `SALARY`, `DUTY_OF_CARE`, or `STAFF_REPORT`.
+Additionally, there are a number of pre-populated users (`Alice`, `Bob`, and `Eve`), resources (the pair of files `/data/local-data-store/employee_file{0,1}.avro`) and rules.
+All examples here are therefore written to be compliant with this example pre-populated data.
+
+## Sample Extract
+```shell script
+no-one@disconnected> connect pal://192.168.49.2:30094/?userid=Alice
+Connected to pal://192.168.49.2:30094/?userid=Alice
+
+Alice@192.168.49.2> register purpose=SALARY file:/data/local-data-store/
+6b6784e4-2383-40ac-92dc-7daa3b0a67b4
+
+Alice@192.168.49.2> cd 6b6784e4-2383-40ac-92dc-7daa3b0a67b4
+Selected 6b6784e4-2383-40ac-92dc-7daa3b0a67b4
+
+Alice@192.168.49.2#6b6784e4-2383-40ac-92dc-7daa3b0a67b4> ls
+file:/data/local-data-store/employee_file1.avro
+file:/data/local-data-store/employee_file0.avro
+
+Alice@192.168.49.2#6b6784e4-2383-40ac-92dc-7daa3b0a67b4> cat file:/data/local-data-store/employee_file0.avro
+Objavro.schema`{"type":"record","name":"Employee","namespace":"uk.gov.gchq.synt[+ 18086 characters]
+```
+
+
+## Commands
```

Comment:
"List of Commands" help me know that this is a heading with the following are sub-headings